src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -635,9 +652,18 @@ void DistractedSequenceRecallTestNetwork()
  * Train the specified networks on the Derek D. Monner's distracted sequence
  * recall task.
  */
-BOOST_AUTO_TEST_CASE(DistractedSequenceRecallTest)
+BOOST_AUTO_TEST_CASE(LSTMDistractedSequenceRecallTest)
+{
+  DistractedSequenceRecallTestNetwork<LSTM<>>();
+}
+
+/**
+ * Train the specified networks on the Derek D. Monner's distracted sequence
+ * recall task.
+ */
+BOOST_AUTO_TEST_CASE(GRUDistractedSequenceRecallTest)
 {
-  DistractedSequenceRecallTestNetwork();
+  DistractedSequenceRecallTestNetwork<GRU<>>();
```

Comment:
I like the idea to reuse the existing test code, but can we tune the GRU parameter to accelerate the test, or do we require the same settings to get stable results. The problem I see is are the current runtimes on the sparc machine: http://masterblaster.mlpack.org/job/mlpack%20-%20nightly%20matrix%20build/arch=sparc64,armadillo=armadillo-5.500.2,buildmode=debug/lastCompletedBuild/testReport/mlpackTest/RecurrentNetworkTest/

I thought the latest change improved the timings, but it looks like that's not true at least for the sparc machines.